frontend/packages/console-shared/src/selectors/common.ts

```diff
@@ -1,10 +1,12 @@
 import * as _ from 'lodash';
-import { K8sResourceCommon } from '@console/internal/module/k8s';
+import { K8sResourceCommon, K8sResourceKind } from '@console/internal/module/k8s';
 
 export const hasLabel = (obj: K8sResourceCommon, label: string): boolean =>
   _.has(obj, ['metadata', 'labels', label]);
 export const getName = <A extends K8sResourceCommon = K8sResourceCommon>(value: A) =>
   _.get(value, 'metadata.name') as K8sResourceCommon['metadata']['name'];
+export const getStatus = <A extends K8sResourceKind = K8sResourceKind>(value: A) =>
+  _.get(value, 'status') as K8sResourceKind['status'];
```

Comment:
I'm not really a fan of these selectors... I would just use `node?.status` directly instead of adding a new one.

Reply:
sure.